src/composable/ConditionalOrder.ts

```diff
@@ -293,23 +295,30 @@ export abstract class ConditionalOrder<D, S> {
     return composableCow.callStatic.singleOrders(owner, this.id)
   }
 
+  /**
+   * Checks the value in the cabinet for a given owner and chain
+   *
+   * @param owner The owner of the conditional order.
+   * @param chain Which chain to use for the ComposableCoW contract.
+   * @param provider An RPC provider for the chain.
+   * @returns true if the owner authorized the order, false otherwise.
+   */
+  public cabinet(owner: string, chain: SupportedChainId, provider: providers.Provider): Promise<string> {
```

Comment:
Parameters are identical to `poll` parameters.
Can it be reused here?

Reply:
Poll params have also the block info, but i think we can make a type, since there's a lot of repetition and is nicer to have named params instead of positional arguments

Reply:
Created a new type `OwnerContext` for this